rts/Lua/LuaVBOImpl.cpp

```diff
@@ -1015,6 +1127,17 @@ size_t LuaVBOImpl::ModelsVBO()
 }
 
 
+/***
+ *
```

Comment:
Fills in `uvec4` vector of data in attribute number `attrID` for each specified unitDefIDs. The instance data in that attribute will contain the offset to bind position matrix in global matrices SSBO and offset to uniform buffer structure in global per unit/feature uniform SSBO (unused for Unit/FeatureDefs), as well as some auxiliary data ushc as draw flags and team index.

```
SInstanceData:
		, matOffset{ matOffset_ }			 // updated during the following draw frames
		, uniOffset{ uniOffset_ }			 // updated during the following draw frames
		, info{ teamIndex, drawFlags, 0, 0 }     // not updated during the following draw frames
		, aux1 { 0u }
```

Reply:
Lets copy these ones for functions below too.

Reply:
I'll simplify the titles

Reply:
What is that `SInstanceData` block supposed to convey?

Reply:
This block unfortunately does not render well inside the function description, I'll have to leave it out unless you have an idea for `@usage` or `@see` referencing source code.

Reply:
> What is that SInstanceData block supposed to convey?

It's supposed to convey the outline of the data inside the instance attribute.

Reply:
Hmm.. unfortunately it doesn't render well. Can it be referenced inside a param?

Reply:
Frankly we should provide examples in separate page. Right now it gives the glimpse of idea how it should be used but it's not enough.

Reply:
Ok, I put inside `@usage`, it's the best thing I found.